tests/lib/assertions.py

```diff
@@ -1,12 +1,15 @@
-from .constants import CORS_METHODS, CORS_MAX_AGE, CORS_ALLOW_HEADERS, CORS_EXPOSE_HEADERS, CORS_POLICY, UNEXPECTED_429
+from .constants import *
```

Comment:
Sonar won't like this, if you want access to all constants do `import lib.constants as constants` then access the constants as `constants.CORS_METHODS` etc